include/CXXGraph.hpp

```diff
@@ -1,7 +1,11 @@
 #ifndef __CXXGRAPH_H__
 #define __CXXGRAPH_H__
 
-#include "CXXGraphConfig.h"
+// the configured options and settings for CXXGraph
```

Comment:
Maybe this file should not exist anymore, if generated when the cmake is called.